test/common/3scale_crudl.go

```diff
@@ -37,17 +37,23 @@ func Test3ScaleCrudlPermissions(t *testing.T, ctx *TestingContext) {
 	keycloakHost := rhmi.Status.Stages[v1alpha1.AuthenticationStage].Products[v1alpha1.ProductRHSSO].Host
 	redirectUrl := fmt.Sprintf("%v/p/admin/dashboard", host)
 
-	tsClient := resources.NewThreeScaleAPIClient(host, keycloakHost, redirectUrl, ctx.HttpClient, ctx.Client, t)
+	httpClient, err := NewTestingHTTPClient(ctx.KubeConfig)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	tsClient := resources.NewThreeScaleAPIClient(host, keycloakHost, redirectUrl, httpClient, ctx.Client, t)
 
 	// Login to 3Scale
-	err = loginToThreeScale(t, host, threescaleLoginUser, DefaultPassword, "testing-idp", ctx.HttpClient)
+	err = loginToThreeScale(t, host, threescaleLoginUser, DefaultPassword, "testing-idp", httpClient)
 	if err != nil {
 		dumpAuthResources(ctx.Client, t)
 		// t.Fatalf("[%s] error ocurred: %v", getTimeStampPrefix(), err)
 		t.Skipf("flakey test [%s] error ocurred: %v jira https://issues.redhat.com/browse/MGDAPI-557 ", getTimeStampPrefix(), err)
 	}
 
 	// Make sure 3Scale is available
+	t.Log("testing 3scale is available")
```

Comment:
Would it make more sense to have the logs inside the `if err != nil` as there is no point in adding the logs if the stage passes. This would the same for other `t.Log()'s ` you have added.

Reply:
Yeah, makes sense. Updated :+1: